projects/archiver/src/eventTask.ts

```diff
@@ -26,10 +28,57 @@ export const extractError = (error?: { Cause: string }): string | undefined => {
     }
 }
 
+interface Topic {
+    type: string
+    name: string
+}
+
+interface ScheduleAppStoreNotificationPayload {
+    id: string
+    type: string
+    topic: Topic[]
+    key: string
+    name: string
+    date: string
+    sender: string
+}
+
+const scheduleAppStoreNotification = (
+    issue: Issue,
+    issuePublication: IssuePublication,
+): any => {
+    console.log('scheduleAppStoreNotification()')
```

Comment:
This is a slightly odd log format 🤔

Reply:
it will go away, that is only for skeleton